Assets/Python/CityNameManager.py

```diff
@@ -5626,14 +5703,82 @@ def onRevolution(iCiv):
 #Language: Korean
 {
 	"Beijing"		:	"Bukgyeong",
-	"Zhongdu"		:	"Bukgyeong",
+	"Zhongdu"		:	"Bukpyeong",

```

Comment:
There is still a typo here.

Reply:
that's not a typo, that's the korean pronunciation of Beiping, the name of Beijing in the Ming era.
Since Beijing has two separate names that change through eras in-game,
I separated the Korean names as well.

Reply:
But apparently Bukpyeong does not map to Zhongdu? I don't think it makes sense to handle names this way. It would be different if Beiping existed as a name but a name that does not exist for the native language should not be reflected in another language.

Reply:
Ok, I'll revert Bukpyeong to Beijing.